src/main/resources/Diagram/WebHome.xml

```diff
@@ -64,14 +68,14 @@
 {{/velocity}}
 
 {{velocity}}
-#set ($diagramClassReference = $services.model.createDocumentReference('', 'Diagram', 'DiagramClass'))
-#if ($services.licensing.licensor.hasLicensureForEntity($diagramClassReference))
+#if ($services.licensing.licensor.hasLicensureForEntity($services.model.createDocumentReference('', 'Diagram', 'DiagramClass')))
+  #selectDiagramsRequiringMigration()
   #if ($toBeMigratedEntries &amp;&amp; $toBeMigratedEntries.size() &gt; 0)
+    #if ($toBeMigratedEntries.size() &gt; 1)#set ($label = 'diagrams require')#else#set ($label = 'diagram requires')#end
```

Comment:
Using a translation keys like this:

    key={0} {0,choice,1#diagram requires|1<diagrams require} a migration. Please click {1}here{2} to proceed.

is more elegant.

Reply:
Indeed, thanks for the tip.